ffi/maybe.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE.md for licensing terms.
+
+package ffi
```

Comment:
Let's add a reference to the avalanchego implementation.

While I'd love to use that, we can't create.a circular reference. Perhaps we can create an issue in avalanchego to move these into a shareable subproject, and we can depend on that.